server/queues/processors/NotificationsProcessor.ts

```diff
@@ -237,6 +249,23 @@ export default class NotificationsProcessor extends BaseProcessor {
       return false;
     }
 
+    // Deliver only a single notification in a 12 hour window
+    const notification = await Notification.findOne({
```

Comment:
I'm tempted to suggest we abstract the entire logic into a configuration on the email class so you can define the suppression rules per-email. But it's probably over engineering in this moment as I'm not sure if any other types of email will need similar logic to this, document edits is kind of a special case.

Reply:
Actually, it occurred to me as well. My suggestion is that we place the suppression rules as it is for now since we don't have many yet. The `shouldNotify` is good enough, since it's explicit and more importantly _keeps all suppression rules in one place_. Not in favor of detouring yet. Makes more sense once we have a clear distinction of suppression rules based on notification types.